meta-lmp-base/recipes-security/optee/optee-os-lmp.inc

```diff
@@ -0,0 +1,9 @@
+# LmP common include for optee_os
+
+# Also deploy optee_os config used during build
+do_deploy:append() {
+    install -D -m 644 ${B}/conf.mk ${DEPLOYDIR}/${PN}-config-${MACHINE}-${PV}-${PR}
+    cd ${DEPLOYDIR}
```

Comment:
Since it is just 2 lines, avoid 'cd' and just link using ${DEPLOYDIR} on both lines.

Reply:
Done.